x-pack/plugins/cases/common/ui/types.ts

```diff
@@ -103,6 +103,15 @@ export interface ResolvedCase {
   aliasPurpose?: ResolvedSimpleSavedObject['alias_purpose'];
 }
 
+export interface Attachment {
```

Comment:
To avoid confusion with other types of attachments (comments) we can rename it to `File`

Reply:
I ended up removing these actually. There is already a `FileJSON` type from `shared-ux` that has what we need.